root/view/src/main/java/at/fhv/itb5c/view/user/UserViewController.java

```diff
@@ -83,9 +85,14 @@
 	public UserViewController(UserModel userModel, UserViewState initialiseState) {
 		_userModel = userModel;
 		_userViewStates = new HashMap<>();
-
+		
+		RouteProvider.getInstance().add(DetailUserViewControlls.class, "/view/fxml/DetailUserViewControlls.fxml");
+		RouteProvider.getInstance().add(NewUserViewControllsController.class, "/view/fxml/NewUserViewControlls.fxml");
+		RouteProvider.getInstance().add(ModifyUserViewControlls.class, "/view/fxml/ModifyUserViewControlls.fxml");
+		
```

Comment:
These lines are called whenever a new `UserViewController` is created. Wouldn't it be enough if the routes where registered once?